qtfred/src/ui/dialogs/ShipEditor/ShipCustomWarpDialog.h

```diff
@@ -24,14 +24,30 @@ class ShipCustomWarpDialog : public QDialog {
 	ShipCustomWarpDialog(QDialog* parent, EditorViewport* viewport, bool departure, int wingIndex, bool wingMode);
 	~ShipCustomWarpDialog() override;
 
+	void accept() override;
+	void reject() override;
+
   protected:
 	/**
 	 * @brief Overides the Dialogs Close event to add a confermation dialog
 	 * @param [in] *e The event.
 	 */
 	void closeEvent(QCloseEvent*) override;
-
-  private:
+  private slots:
+	void on_buttonBox_accepted();
+	void on_buttonBox_rejected();
+	void on_comboBoxType_currentIndexChanged(int);
+	void on_lineEditStartSound_editingFinished();
+	void on_lineEditEndSound_editingFinished();
+	void on_doubleSpinBoxEngage_valueChanged(double);
+	void on_doubleSpinBoxSpeed_valueChanged(double);
+	void on_doubleSpinBoxTime_valueChanged(double);
+	void on_doubleSpinBoxExponent_valueChanged(double);
+	void on_doubleSpinBoxRadius_valueChanged(double);
+	void on_lineEditAnim_editingFinished();
+	void on_checkBoxSupercap_toggled(bool);
+	void on_doubleSpinBoxPlayerSpeed_valueChanged(double);
+  private: //NOLINT
```

Comment:
Really minor nitpick but these spots could use a more targeted no lint as I've used elsewhere. ` // NOLINT(readability-redundant-access-specifiers)`

But it really doesn't matter.. just a same-ness thing.

Reply:
Done